Assets/Code/VehicleEditor/EditSystem.cs

```diff
@@ -307,4 +375,168 @@ partial class EditSystem : SystemBase {
     protected override void OnDestroy() {
         input.Disable();
     }
+
+    
+    void ShowPathToRoot(ref EntityCommandBuffer actionQueue , Entity parentPart)
+    {
+        // this is just some debug stuff to see how the parts where created
+        
+            Debug.Log("");
+            int iteration = SystemAPI.GetComponent<Part>(parentPart).layer; // ok idk if iteration is the right term but it will do just fine for now
+
+            int[] symmetryCounts = new int[iteration];
+            int[] IDs = new int[iteration];
+            Entity[] entitiesRootToBottom = new Entity[iteration]; // technicly later on this will be a root to bottom
+
+            var currentPart = parentPart;
+            for (int i = 0; i < iteration; i++)
+            {
+                var siblings = EntityManager.GetBuffer<Siblings>(currentPart);
+                symmetryCounts[iteration - i - 1] = siblings.Length;
+                IDs[iteration - i - 1] = SystemAPI.GetComponent<Part>(currentPart).id;
+                entitiesRootToBottom[iteration - i - 1] = currentPart;
+
+                if (i != iteration - 1) { currentPart = SystemAPI.GetComponent<Parente>(currentPart).Value; } // root has no parent
+            }
+            foreach (var t in entitiesRootToBottom)
+            {
+                Debug.Log(t.Index);
+                LocalTransform lt = SystemAPI.GetComponent<LocalTransform>(t);
+                if (lt.Scale == 1.0f) { lt.Scale = 1.2f; }
+                else { lt.Scale = 1.0f; }
+                actionQueue.SetComponent(t, lt);
+            }
+
+
+        
+
+
+    }
+
+    // dont know if not using ref will be an issue
+    // THERE MIGHT BE AN ISSUE , IF A PART IS REGISTERED AS ANOTHER PART SIBLING BUT THEN THIS FIRST PART IS DESTROYED
+    // IF A NEW PART IS PLACED WITH THE SAME VALUE AS THE DESTROYED ONE THERE MIGHT BE SOME ISSUE -> to fix this the deleted part's reference should be cleaned from his siblings and parents
+    void RootSymmetryMode(EntityCommandBuffer actionQueue,Entity part,Entity parentPart,int symmetryCount,LocalTransform placeTransform,int MaxIt) {
```

Comment:
should remove unused arguments here

(note that EntityCommandBuffer is a struct, which means it's passed by value like a number, which means you have to either use `ref` or return it if you make changes to it in this method as otherwise you'll lose the changes)